selenium_tests/cross_listing/page_objects/cross_listing_main_page.py

```diff
@@ -35,11 +38,19 @@ def ERROR_TEXT_LOCATOR(cls, expected_text):
 class MainPageObject(CrossListingBasePageObject):
     page_loaded_locator = Locators.HEADING_ELEMENT
 
+    # todo: consider moving into the BasePageObject, with override flexibility
+    def __init__(self, driver):
```

Comment:
Understood how you are using it, but not understanding the __init__reference here. Will ping you later. :)